src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ReconfigureFundingCycleSettingsPage/hooks/useReconfigureFundingCycle.ts

```diff
@@ -120,6 +133,16 @@ export const useReconfigureFundingCycle = ({
         mustStartAtOrAfter: editingMustStartAtOrAfter,
       }
 
+      if (hasConflictingNftTx) {
+        reconfigureFundingCycleData = mergeAlreadyQueuedNftTx(
+          reconfigureFundingCycleData,
+        )
+      } else if (hasConflictingEditCycleTx) {
+        reconfigureFundingCycleData = mergeAlreadyQueuedEditCycleTx(
+          reconfigureFundingCycleData,
+        )
+      }
```

Comment:
@johnnyd-eth might I suggest: could we have a single hook - maybe `useResolveCycleEditConflicts` - that does all this logic internally? This way if theres a bug, we have a single source to fix rather than multiple.

Reply:
Yeah good call, can do that